subjects/java/checkpoints/time-tracker/README.md

```diff
@@ -0,0 +1,72 @@
+## TimeTracker
+
+### Instructions
+
+Write a `ProjectTime` class to track the duration of a project using start and end times. You will be asked to add getter and setter methods for the `startTime` and `endTime` attributes. When you set one of them, you must update the `hoursLogged` attribute. The `hoursLogged` getter should output the hours in the following format:
```

Comment:
What is `hoursLogged`? Is it supposed to be `endTime` - `startTime`? It would be nice to have more information about what's expected from `hoursLogged` (even if it's possible to understand from the given examples)

Reply:
I don't think that this need more information, because its clearly mentioned that they have to track duration, and there is only one way to calculate duration.